numba_dppy/tests/_helper.py

```diff
@@ -88,15 +88,15 @@ def is_windows():
 skip_windows = pytest.mark.skipif(is_windows(), reason="Skip on Windows")
 
 skip_no_opencl_gpu = pytest.mark.skipif(
-    not has_opencl_gpu(),
+    not has_opencl_gpu() and config.TESTING_SKIP_NO_OPENCL_GPU,
     reason="No opencl GPU platforms available",
 )
 skip_no_opencl_cpu = pytest.mark.skipif(
-    not has_opencl_cpu(),
+    not has_opencl_cpu() and config.TESTING_SKIP_NO_OPENCL_CPU,
     reason="No opencl CPU platforms available",
 )
 skip_no_level_zero_gpu = pytest.mark.skipif(
-    not has_level_zero(),
+    not has_level_zero() and config.TESTING_SKIP_NO_LEVEL_ZERO_GPU,
```

Comment:
`has_level_zero()` is supposed to skip a test when there is no level zero device already, why do we need this? Not clear from the PR description.